launch/app/app.bndrun

```diff
@@ -123,8 +123,6 @@ feature.openhab-model-runtime-all: \
 # done
 #
 -runbundles: \
-	org.ops4j.pax.logging.pax-logging-api;version='[2.2.3,2.2.4)',\
-	org.ops4j.pax.logging.pax-logging-log4j2;version='[2.2.3,2.2.4)',\
```

Comment:
IIRC the Pax Logging bundles are at the top so those bundles are started first so the log levels are applied immediately which prevents noisy debug/trace logging at startup.

Reply:
I'll have a look. I was thought not to touch whatever the resolver creates ;-)